src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
```

Comment:
Here we have an element of another element:
`An element belongs to a block, not to another element. That's why the prefix — is the name of a block, not the name of another element.`

use name for style like this `navigation__your-style-name` eg. `navigation__list-item`